CampusTour/TableCells/ItemOfInterestTableViewCell.swift

```diff
@@ -0,0 +1,58 @@
+//
+//  ItemOfInterestTableViewCell.swift
+//  CampusTour
+//
+//  Created by Serge-Olivier Amega on 3/18/18.
+//  Copyright © 2018 cuappdev. All rights reserved.
+//
+
+import UIKit
+
+class ItemOfInterestTableViewCell: UITableViewCell {
+    static let reuseId = "ItemOfInterestTableViewCell"
+    
+    struct Model {
+        let title: String
+        let date: Date
+        let description: String
+    }
+    
+    var rootStackView: UIStackView?
+    
+    func headerView(model: Model) -> UIView {
+        let header = UIStackView()
+        header.axis = .horizontal
+        
+        let titleLabel = UILabel()
+        titleLabel.text = model.title
+        header.addArrangedSubview(titleLabel)
+        
+        let dateLabel = UILabel()
+        dateLabel.text = model.date.description //TODO format better
+        header.addArrangedSubview(dateLabel)
+        
+        return header
+    }
+    
+    func setCellModel(model: Model) {
+        rootStackView?.removeFromSuperview()
```

Comment:
are you doing this instead of doing prepareforreuse?

Reply:
when prepareforreuse gets called I can't give the cell a model